modules/kn-trigger-delete.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-kn-trigger.adoc
+
+[id="kn-trigger-delete"]
+= Deleting a trigger using `kn`
```

Comment:
I see that you changed `kn` to the Knative CLI, just wondering if that should be done here as well?